changelog/fix-minor-readability-change-in-test

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: fix
```

Comment:
The type is rather `dev`, can you recreate the changelog entry, please?

Reply:
Thanks for pointing that out, @kalessil . I have amended the type to `dev`.  I also re-read the description of the `Significance` field  and thought `minor` would be more appropriate, since the PR does not have any bug fix per-se. I hope that is fine.

Reply:
Thanks @nagpai !